### PR TITLE
Include image tag to rss feed

### PR DESCRIPTION
**Pull request by Shilario**
### Trello Ticket

* https://trello.com/c/1lniUqyV/588-articles-feeds-generate-rss-feed-for-samsung

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -135,6 +150,17 @@ function sendError(res, e, message = e.message) {
  * @return {Promise}
  */
 function render({ feed, meta, attr }, info, res) {
+  if (feed.length) {
+    let imageIndex = findIndexOfElementInArray(feed[0], 'image'),
+      url = _get(feed[0][imageIndex].image, 'url');
```
Comment (PedroAlbR) [inline index.js]:
```suggestion
      url = _get(feed[0][imageIndex], 'image.url');
```

File: index.js [hunk 2]
```diff
@@ -135,6 +150,17 @@ function sendError(res, e, message = e.message) {
...
+
+    if (url) {
+      meta.image = {
```
Comment (PedroAlbR) [inline index.js]:
```suggestion
      meta.image = { url }
```

File: index.js [hunk 3]
```diff
@@ -153,6 +179,33 @@ function render({ feed, meta, attr }, info, res) {
     .catch(e => sendError(res, e));
 }
 
+/**
+ * Finds the index of a given element in an array.
+ *
+ * @param {Array} array
+ * @param {String} element
```
Comment (PedroAlbR) [inline index.js]:
```suggestion
 * @param {string} element
```

File: index.js [hunk 4]
```diff
@@ -153,6 +175,33 @@ function render({ feed, meta, attr }, info, res) {
...
+ * @returns {number}
+ */
+function findIndexOfElementInArray(array, element) {
+  return _findIndex(array, (item) => item[element]);
```
Comment (jctaveras) [inline index.js]: I think we can afford this one by just using vanilla JS

```suggestion
  return array.findIndex((item) => Boolean(item[element]));
```

File: index.js [hunk 5]
```diff
@@ -153,6 +175,33 @@ function render({ feed, meta, attr }, info, res) {
...
+ * @param {String} url
+ * @param {String} link
+ * @param {String} title
+ * @returns {Object}
+ */
+function formatImageTag(url, link, title) {
+  return { image: [
+    { url },
+    { link },
+    { title}
```
Comment (jctaveras) [inline index.js]: Missing space here
```suggestion
    { title }
```

File: index.js (hunk 4 above)
Comment (amirandalibi) [reply index.js]: +1

**Pull request merged**